tests/MoBi.Tests/IntegrationTests/ProjectTaskIntegrationTests.cs

```diff
@@ -39,15 +52,57 @@ public void should_have_set_the_name_of_the_project_to_the_name_of_the_file()
       [Observation]
       public void loaded_building_blocks_should_each_be_contained_in_their_own_module()
       {
-         _context.CurrentProject.Modules.Count.ShouldBeEqualTo(8);
-         _context.CurrentProject.Modules.Each(x => x.BuildingBlocks.Count.ShouldBeEqualTo(1));
+         _context.CurrentProject.Modules.Count.ShouldBeEqualTo(1);
+         _context.CurrentProject.Modules.Each(x => x.BuildingBlocks.Count.ShouldBeEqualTo(8));
       }
 
       [Observation]
-      public void each_module_should_be_named_for_the_building_block_and_building_block_type()
+      public void simulations_should_use_modules_from_the_project_to_build_configurations()
       {
-         _context.CurrentProject.Modules.Each(x => x.Name.Contains(x.BuildingBlocks[0].Name).ShouldBeTrue());
-         _context.CurrentProject.Modules.Each(x => x.Name.Contains(_objectTypeResolver.TypeFor(x.BuildingBlocks[0]).Replace("Building Block", string.Empty)).ShouldBeTrue());
+         _context.CurrentProject.Simulations.Each(matchSimulationAndProjectModules);
+      }
+
```

Comment:
remove extra space

Reply:
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/108261271/89f5a1ed-1b57-4d62-bef2-d9e510e4b9dc)
I think there is no extra space here, just normal spacing between methods.
Could you confirm?

Reply:
oh yeah. just in the PR, not in the file.